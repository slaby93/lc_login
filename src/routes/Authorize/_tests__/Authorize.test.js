import React from 'react';
import { shallow } from 'enzyme';
import AuthorizeStyled, { Authorize } from './../Authorize';

describe('<Authorize />', () => {
  test('CSS', () => {
    const mountedComponent = shallow(<AuthorizeStyled
      location={{}}
      match={{ url: 'test1' }}
    />);
    expect(mountedComponent).toMatchSnapshot();
  });

  test('HTML', () => {
    const mountedComponent = shallow(<Authorize
      location={{}}
      match={{ url: 'path' }}
      className="test"
    />);
    expect(mountedComponent).toMatchSnapshot();
  });
});
