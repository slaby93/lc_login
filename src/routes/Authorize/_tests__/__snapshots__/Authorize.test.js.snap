// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Authorize /> CSS 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  margin-top: 5%;
  max-height: 100vh;
  overflow: hidden;
}

@media (min-height:px) {
  .c0 {
    margin-top: 10%;
  }
}

@media screen and (orientation:landscape) {
  .c0 {
    margin-top: 5%;
  }
}

<Authorize
  className="c0"
  location={Object {}}
  match={
    Object {
      "url": "test1",
    }
  }
/>
`;

exports[`<Authorize /> HTML 1`] = `
<div
  className="test"
>
  <Authorize__ShadowWrapper>
    <Switch
      location={Object {}}
    >
      <Route
        component={[Function]}
        exact={true}
        path="path"
      />
    </Switch>
  </Authorize__ShadowWrapper>
</div>
`;
